ci-droid-tasks-consumer-autoconfigure/src/main/resources/application.yml

```diff
@@ -69,9 +69,12 @@ cidroid-behavior:
     limitInDays: 180
 #  whenPullRequestEvent:
   bestPracticeNotifier.enabled: true
+  maxFilesInPRNotifier.enabled: true
   patternToResourceMapping:
 # example of associating a document with a file pattern
     "**/*Dto.java": "https://some_link_to_some_best_practices_repo/noDtoClass.md"
+    maxFilesInPr: 10
```

Comment:
these properties should be under maxFilesInPRNotifier, as they apply to it only. see how PullRequestCleaningHandler is configured with `closeOldPullRequests`, I think it's a good example : there's the `enabled` property, and other properties pertaining to it are nested under `closeOldPullRequests`. I think we should do the same with PullRequestSizeCheckHandler and maxFilesInPRNotifier property.

Reply:
Corrected the configuration